components/automate-gateway/gateway/middleware/authv2/authv2.go

```diff
@@ -82,21 +80,9 @@ func (c *client) Handle(ctx context.Context, subjects []string, projectsToFilter
 	return auth_context.NewContext(ctx, subjects, projects, resource, action), nil
 }
 
-type resp struct {
-	ctx        context.Context
-	authorized bool
-}
-
-func (r *resp) Ctx() context.Context {
-	return r.ctx
-}
-
-func (r *resp) GetAuthorized() bool {
-	return r.authorized
-}
-
+// IsAuthorized takes care of HTTP authorization in the custom HTTP handlers
 func (c *client) IsAuthorized(ctx context.Context, subjects []string, resource, action string,
-	projectsToFilter []string) (middleware.AuthorizationResponse, error) {
+	projectsToFilter []string) (context.Context, bool, error) {
```

Comment:
Inlined the response, a simplification available now that we don't have to unify two different PB responses that both have a `GetAuthorized` method.

Reply:
I would find it helpful to have these returned values be named (`isAuthorized bool`)